### ISSUE TITLE
missing dist directory after npm install

### ISSUE THREAD

**Issue by dsilvasc** [softprops/serverless-localhost#12]
## 🐛 Bug description
After installing this package with npm, the `node_modules/serverless-localhost/` directory contains just `package.json`, `README.md`, `LICENSE`, and `CHANGELOG.md`.  The dist directory is missing.

Consequently running `serverless localhost` fails with
> Serverless plugin "serverless-localhost" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.

I think you need to commit the dist directory to the repository per https://stackoverflow.com/questions/46619788/npm-install-from-a-git-repo-leaves-a-pretty-blank-package

#### 🤔 Expected Behavior
The dist directory should be present after installing.

#### 👟 Steps to reproduce
```
$ npm install softprops/serverless-localhost
$ ls node_modules/serverless-localhost/
CHANGELOG.md  LICENSE       README.md     node_modules/ package.json
$ serverless localhost

  Serverless Error ---------------------------------------

  Serverless plugin "serverless-localhost" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.
```

#### 🌍 Your environment
```
$ npm --version
6.5.0
$ node --version
v11.6.0
$ uname -a
Darwin localhost 18.7.0 Darwin Kernel Version 18.7.0: Sun Dec  1 18:59:03 PST 2019; root:xnu-4903.278.19~1/RELEASE_X86_64 x86_64
```
serverless version: 1.60.4 (standalone)
serverless-localhost version: latest master, 0.1.0

**Comment by y2kappa** [comment]
thanks for this, maybe you should submit a PR?

**Comment by dsilva** [comment]
@y2kappa good point -- submitted https://github.com/softprops/serverless-localhost/pull/15
